Admin/Extension/LockExtension.php

```diff
@@ -57,7 +58,7 @@ public function configureFormFields(FormMapper $form)
                 return;
             }
 
-            $form->add($fieldName, 'hidden', array(
+            $form->add($fieldName, HiddenType::class, array(
```

Comment:
please do not use the `::class` constant and make it compatible with SF <2.8

like here for example: https://github.com/sonata-project/SonataPageBundle/pull/831/files